cli/cmd/lql.go

```diff
@@ -310,8 +296,22 @@ func inputQueryFromURL(url string) (query string, err error) {
 	return
 }
 
-func inputQueryFromEditor(action string, language string) (query string, err error) {
+func inputQueryFromEditor(action string) (query string, err error) {
 	var queryTextTemplate string
+	language := "LQL"
+	if action == "create" {
+		languageSelect := &survey.Select{
+			Message: "Choose query language to create: ",
+			Options: []string{
+				"LQL",
+				"Rego",
```

Comment:
As much as I like the use of Survey here, what will happen if users select Rego but they are not part of the Private Preview customers that have Rego enabled in the backend?

Reply:
There is a feature flag `lpp_rego_enabled` telling whether a customer can use Rego or not. However, it seems we cannot call `GET /api/v2/FeatureFlags/:tag` endpoint for FF check in a public repo like https://github.com/lacework/go-sdk

Reply:
> what will happen if users select Rego but they are not part of the Private Preview customers that have Rego enabled in the backend?

They should get a suitable 400 error.